docs/reference.md

```diff
@@ -711,12 +709,12 @@ time in seconds since the Unix epoch) - and more, depending upon the provider us
 will be returned. Otherwise, the return value will be `null`.
 
 ```js
-var authData = $scope.authObj.$getAuth();
+var firebaseUser = $scope.authObj.$getAuth();
```

Comment:
So the `getCurrentUser()` is what it is called in the new Firebase `3.x.x` SDK. We chose to keep the `$getAuth()` name in AngularFire although we may have been better off using `$getCurrentUser()`. I added [a note](https://github.com/firebase/angularfire/issues/764#issuecomment-224369132) to #764 to consider this change.